demo/sensor_demo.py

```diff
@@ -14,6 +14,16 @@
 start = pd.to_datetime('2022-01-01 12:00:00')
 end = pd.to_datetime('2022-01-01 13:00:00')
 
+# Specify the timezone (location) of where you are simulating
+# (this assumes that the start and end parameters are not already in local time)
+time_zone = 'America/New_York'
```

Comment:
Despite offline conversations about this, do we want to specify this by default? Or do we want to leave it blank to allow the user to manually declare if needed/desired? If the latter, then maybe it would make sense to either def an argument for the user to declare the time zone in the function call, which would then be passed throughout the code where ever its invoked. Or alternatively, if alerting a time zone is critical, it may make sense to just return a message to the user to think about the time zone for their application, but stop short of doing anything about it. Just a thought.

@rykerfish - what do you think about this?